src/SFA.DAS.EAS.Web/Views/EmployerCommitments/Acknowledgement.cshtml

```diff
@@ -4,7 +4,7 @@
     <div class="column-two-thirds">
 
 
-        <div class="govuk-box-highlight">
+        <div class="govuk-box-highlight green-box">
```

Comment:
In the rest of the AML journey, all error summaries and green flash messages sit in a column-full so are 100% width of the page